pkgs/top-level/php-packages.nix

```diff
@@ -545,11 +545,8 @@ lib.makeScope pkgs.newScope (self: with self; {
         {
           name = "sockets";
           doCheck = false;
-          patches = lib.optional (php.version == "8.1.2")
-            (fetchpatch {
-              url = "https://github.com/php/php-src/commit/07aaa34cd418c44f7bc653fafbf49f07fc71b2bf.patch";
-              sha256 = "sha256-EwVb09/zV2vJ8PuyLpKFCovxe6yKct0UBvishZaordM=";
-            });
+          patches = lib.optional (php.version == "8.1.2" || php.version == "8.0.15")
+            ../development/interpreters/php/Fix-sockets-extension.patch;
```

Comment:
That file does not exist.

Reply:
I know, went for lunch, fixing this asap.

Reply:
Fixed.